### ISSUE TITLE
Error: Failed to install 'brinla': package ‘INLA’ is not available

### ISSUE THREAD

**Issue by fredriklohaa** [julianfaraway/brinla#3]
Dear Prof. Julian Faraway,

I am trying to install your package and am using the command:

with_libpaths(new="C:/Program Files/R/R-3.6.1/library/",install_github("julianfaraway/brinla"))
I get the error message: 

Error: Failed to install 'brinla' from GitHub:
  (converted from warning) package ‘INLA’ is not available (for R version 3.6.1)

Yet I have INLA installed on my computer: This is INLA_19.09.18 built 2019-09-18 17:52:48 UTC.
See www.r-inla.org/contact-us for how to get help.

Do you know what is wrong?

**Comment by julianfaraway** [comment]
Maybe this is because you are not using the install method recommended in the README?

**Comment by rancoper** [comment]
Dear Prof. Julian Faraway @julianfaraway 

Using the install method in the README was not enough to install brinla. Look below the message in the Console:

> remotes::install_github("julianfaraway/brinla")
Downloading GitHub repo julianfaraway/brinla@HEAD
Skipping 1 packages not available: INLA
✓  checking for file ‘/private/var/folders/j3/z0zylnbx7yx2jsdv8bg82gk00000gn/T/Rtmp4RJ8ZL/remotes39ee6e7a60a5/julianfaraway-brinla-338abb5/DESCRIPTION’ ...
─  preparing ‘brinla’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  looking to see if a ‘data/datalist’ file should be added
─  building ‘brinla_0.1.0.tar.gz’
** installing *source* package ‘brinla’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error: (converted from warning) package ‘sp’ was built under R version 3.6.2
Execution halted
ERROR: lazy loading failed for package ‘brinla’
** removing ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/brinla’
Error: Failed to install 'brinla' from GitHub:
  (converted from warning) installation of package ‘/var/folders/j3/z0zylnbx7yx2jsdv8bg82gk00000gn/T//Rtmp4RJ8ZL/file39ee245f294d/brinla_0.1.0.tar.gz’ had non-zero exit status


Do you know what is wrong?

Ps: I also tried with 
library(devtools)  
install_github("julianfaraway/brinla")

**Comment by julianfaraway** [comment]
R is currently at version 4.0.2 but you are using a version of sp built for 3.6.2. You could try updating. Also it is best to install INLA first - the log implies you have not done this yet.

**Comment by sandyherho** [comment]
Prof. @julianfaraway ,
I got the same problem even though I followed exactly the README file and use R version 4.0.2.  Here is the error message:

`Error: (converted from warning) package 'Matrix' was built under R version 4.0.3
Execution halted
ERROR: lazy loading failed for package 'brinla'
removing 'D:/Program Files/R/R-4.0.2/library/brinla'
Error: Failed to install 'brinla' from GitHub:
  (converted from warning) installation of package ‘C:/Users/User/AppData/Local/Temp/RtmpWovSD0/fileb184ebe6742/brinla_0.1.0.tar.gz’ had non-zero exit status` 

Is the brinla package use another version of Matrix package?
Thanks.

**Comment by julianfaraway** [comment]
I do not think this is the same problem and it does not seem to be a problem with brinla. It seems you are using a version of Matrix for 4.0.3 whereas you have 4.0.2 installed. The error appears to result from this mismatch. I suggest you update your version of R to 4.0.3.
